Dockerfile

```diff
@@ -1,27 +1,31 @@
-FROM golang:latest AS builder
+FROM golang:1.12 AS builder
+
+ENV GO111MODULE=on
 
 WORKDIR /go/src/github.com/algolia/sup3rS3cretMes5age
-ADD . .
 
-RUN go get -v 
-RUN CGO_ENABLED=0 GOOS=linux go build  -o sup3rS3cretMes5age .
+ADD . .
 
-RUN go run /usr/local/go/src/crypto/tls/generate_cert.go --host localhost
+RUN go mod download
+RUN CGO_ENABLED=0 GOOS=linux go build -o sup3rS3cretMes5age .
 
-FROM alpine:latest
 
-EXPOSE 1234
+FROM alpine:20210212
 
 ENV \
     VAULT_ADDR \
-    VAULT_TOKEN
+    VAULT_TOKEN \
+    SUPERSECRETMESSAGE_HTTP_BINDING_ADDRESS \
+    SUPERSECRETMESSAGE_HTTPS_BINDING_ADDRESS \
+    SUPERSECRETMESSAGE_HTTPS_REDIRECT_ENABLED \
+    SUPERSECRETMESSAGE_TLS_AUTO_DOMAIN \
+    SUPERSECRETMESSAGE_TLS_CERT_FILEPATH \
+    SUPERSECRETMESSAGE_TLS_CERT_KEY_FILEPATH
 
-RUN \
-apk add --no-cache ca-certificates ;\
```

Comment:
Don't you want to keep this in case https is enabled ?

Reply:
Good catch, thank you! Fixed by https://github.com/algolia/sup3rS3cretMes5age/pull/50/commits/bb77f2ad3e47f4db308ab50e4fadb1ba6f7a7c75